### ISSUE TITLE
Pico Remote Stops Working When Using Turn-Off Setting in Homekit

### ISSUE THREAD

**Issue by spirospz** [thenewwazoo/homebridge-lutron-caseta-leap#104]
Thanks for this plugin it's amazing.

I added a pico remote to simply turn a ring floodlight light on and off using a short press on button 1 and 5. The pico works as expected and there are no problems.

Now I want the light to turn off automatically after a set period and when I enable the turn-off command in homekit (underneath where I add the accessories for the button press) for any length of time the light does not behave as expected.

The light will turn on once from the pico and then it won't turn on again after turning it off. I have added multiple lights to see if it's an issue with the ring and the problem exists among other lights as well.

When I press the pico button, the homebridge log does report the button press but the light doesn't turn on.

When I delete the action for button 1 single press and add it again without the turn-off command it will work perfectly fine, just no auto off.

Now I can use a dummy switch as a timer to shut it off but I figured I would bring this to your attention.

"name": "Lutron",
            "platform": "LutronCasetaLeap",
            "options": {
                "filterPico": false,
                "filterBlinds": true,
                "clickSpeedDouble": "default",
                "clickSpeedLong": "default"
```
Homebridge Native DSM7

Plugin Version homebridge-lutron-caseta-leap v2.4.5-beta0 and it also happens on 2.4.3
* **Homebridge Version v1.6.0
* **Node.js Version v16.16.0
* **Operating System Synology DSM 7.1.1-42962 Update 4

**Comment by thenewwazoo** [comment]
That's super weird! I honestly probably won't have time to repro this, but I suspect it's a bug in HomeKit. Do you know which version of iOS you're running on your hub? The biggest clue for me is that you're seeing the button presses in the log. As I wrote [in this comment](https://github.com/thenewwazoo/homebridge-lutron-caseta-leap/issues/67#issuecomment-1217255270), once the plugin has passed the message to HomeKit, it's up to the hub's internal logic as to what needs to get turned on/off. My suspicion is that the hub isn't recognizing that the scene has been "un-set" by the turn-off logic and so it's refusing to do anything.

**Comment by MoTechnicalities** [comment]
To test, I set up a Pico single-press middle button action to turn ON a light I have running under the Tuya plugin, then to turd OFF automatically after 1 minute.
It worked fine.

Worth noting is that Ring Floodlights are hardwired to the built-in motion sensor. I would never expect expected behavior out of those lights.

On an additional side note, one of my Ring Floodlights started strobing a couple of nights ago. I had to take it down and snip the wire to the lights, leaving only the camera working now. I will now wait for a Matter replacement. If Ring adopts Matter, great but if not then maybe Wyze Cams will and I'll replace it with theirs.

**Comment by spirospz** [comment]
I have three hubs, Apple TV HD running tvOS 16.3.2, HomePod mini running 16.3.2, and an Apple TV 4K running 16.3.2. The Apple TV 4K is always connected and the other two are on standby. I am on the new architecture since it was first available before apple pulled it.

You're right, the ring device isn't a reliable device to test with but when I was testing I had the pico activate the ring flood light and a lutron caseta lamp dimmer which both failed to work when adding the auto-off.

I have another pico in another room in which button 3 single press activates two lutron caseta in-wall dimmers without the auto off and they work fine. After seeing MoTechnicalities post I decided to add a 1-minute auto-off to that action and instead of completely failing, only one device is failing whereas the other does still work with the auto-off enabled. I went back to test it again after a few minutes and both lights did work. The failure seems to happen when switching on and off waiting only a few seconds between on and off commands.

Thanks again for a great plugin. I had a problem and just wanted to report it in case it was plugin related. For now, my workaround is using a dummy switch as a timer.

 Also on a side note, lately as we all know homekit isn't reliable as it used to be. I do have instances when pretty much all of my homebridge plugins work and fail work and fail especially after updating to new firmware that apple releases. I am running a Synology rt2600 with two mr2200 and found that if I delete the two mr2200s and leave the main router running solo and reboot the apple devices everything comes back to life and works. After that, I will add the two mr2200 back to the network and things stay running properly. I saw someone's post online about homekit having issues with mesh routers so I figured I'd give it a try and it works for me. Hope this helps anyone that has been having any issues.
